pomcorn/element.py

```diff
@@ -285,19 +285,23 @@ def select(self, value: str, only_visible: bool = True):
     def click(
         self,
         only_visible: bool = True,
+        scroll_to: bool = True,
         wait_until_clickable: bool = True,
     ):
         """Click on element.
 
         Args:
             only_visible: Flag for viewing visible elements. If this is `True`
                 (default), then this method will only get visible elements.
+            scroll_to: Whether to scroll to element before clicking or not.
             wait_until_clickable: Wait until the element is clickable before
                 clicking, or not (default `True`).
 
         """
         if wait_until_clickable:
             self.wait_until_clickable()
+        if scroll_to:
```

Comment:
I think it's better to change order of `wait` and `click`.
Consider case when button is not visible on a screen (need to scroll) and not clickable.
If `wait` condition will fail, it would be nice to make a screenshot to investigate issue later. But screenshot won't show desired button because scroll not applied yet